.github/workflows/ci.yml

```
@@ -277,6 +277,17 @@ jobs:
         export RUSTDOCFLAGS="${{ matrix.toolchain == 'nightly' && '-Z unstable-options --document-hidden-items' || '' }} $RUSTDOCFLAGS $METADATA_DOCS_RS_RUSTDOC_ARGS"
         ./cargo.sh +${{ matrix.toolchain }} doc --document-private-items --package ${{ matrix.crate }} ${{ matrix.features }}
 
+    - name: Check dead links
+      run: |
+        cargo install -q cargo-deadlinks --version 0.8.1
```

```suggestion
        set -eo pipefail
        cargo install -q cargo-deadlinks --version 0.8.1
```

And honestly, in other jobs (notably, the previous one) while you're at it. We should have it in every job.